tasks/prelim.yml

```diff
@@ -190,3 +190,27 @@
       - RHEL-07-021600
       - RHEL-07-021610
       - RHEL-07-021620
+
+- name: "PRELIM | RHEL-07-010480 RHEL-07-010490 | Install python-passlib for hash creation"
+  yum:
+      name: python-passlib
+  when:
+      - inventory_hostname == 'localhost' or inventory_hostname == '127.0.0.1'
+      - rhel_07_020030 or rhel_07_010480
+
+- name: "PRELIM | RHEL-07-020210 RHEL-07-020220| Install SELinux related dependencies"
+  yum:
+      name: "{{ item }}"
+  with_items:
+      - selinux-policy
```

Comment:
would you ever install selinux-policy without either selinux-policy-targeted or selinux-policy-mls?  I think I've broken a system with only installing selinux-policy, but I could be mis-remembering...

Reply:
This is pulled in as a dep of the other policy packages below.